src/main/drivers/accgyro/accgyro_spi_icm20649.c

```diff
@@ -35,8 +35,8 @@
 #include "accgyro_mpu.h"
 #include "accgyro_spi_icm20649.h"
 
-static bool use4kDps = true; // TODO: make these configurable for testing
-static bool use30g = true;
+#include "sensors/acceleration.h"
+#include "sensors/gyro.h"
```

Comment:
You are adding an upwards dependency here. Driver code should not depend on sensor code. Instead add an extra field in `gyroDev_t` and set it from gyroconfig in `gyroInitSensor` in `sensors/gyro.c` before `gyroDetect` is called. Similarly for acc.